python/ray/_private/test_utils.py

```diff
@@ -209,6 +209,8 @@ def run_string_as_driver(driver_script: str, env: Dict = None):
     Returns:
         The script's output.
     """
+    import copy
+    env = copy.deepcopy(os.environ).update(env or {})
```

Comment:
What does this block do?

Reply:
We need to pass bootstrap env to popen. I thought it's passed automatically, but from my local test somehow it's not.